packages/kotti-ui/source/kotti-table/KtTable.vue

```diff
@@ -96,11 +90,6 @@ export default {
 		expandMultiple: { default: false, type: Boolean },
 		selected: { default: () => [], type: Array },
 	},
-	inject: {
```

Comment:
This seems like it could potentially break the table. Why was this done and why is it safe?

Reply:
Was confused why that happened, as I didn't remember doing this change. I reintroduced it and eslint wanted me to change positions of `provide`. Not sure what happened to remove those lines. Good that you noticed it.

Reply:
@HansAuger was this an eslint auto fixable issue? If yes, then this kind of thing can happen when you save twice in quick succession due to some VSCode race conditions

Reply:
Definitely autofixable, just reproduced it.